server/verify.go

```diff
@@ -321,54 +325,68 @@ func makePool(certs []*x509.Certificate) *x509.CertPool {
 // VerifyGceTechnology checks the GCE-specific GceNonHost event's Trusted Execution Technology (TEE)
 // claim using attestation reports if the technology supports them, and only then validates that a
 // particular technology has proven that it is in use.
-func VerifyGceTechnology(attestation *pb.Attestation, tech pb.GCEConfidentialTechnology, opts *VerifyOpts) error {
+func VerifyGceTechnology(attestation *pb.Attestation, tech pb.GCEConfidentialTechnology, opts *VerifyOpts) (*pb.MachineState, error) {
```

Comment:
This should go in `parsePCClientEventLog` since it depends on the resulting state. Long term, we will have branching to attempt to get a full MachineState from TeeAttestation. If successful, we shouldn't need to read TPM-related data. If unsuccessful (e.g., TDX machine without RTMR support or SNP machine right now), then we merge in TPM MachineState.

Reply:
Discussed offline, added a wrapper function around `parsePCClientEventLog` and `VerifyGceTechnology`